batch/batch_commands.go

```diff
@@ -83,8 +83,10 @@ func ListBatchJobs(c *cli.Context) error {
 // BatchTerminate terminate a list of workflows
 func BatchTerminate(c *cli.Context) error {
 	operator := common.GetCurrentUserFromEnv()
+	rps := float32(c.Float64(common.FlagRPS))
```

Comment:
AFAICT, the CLI library doesn't have a built-in `Float32` flag, so we need to downcast here. Which is fine; the required precision we expect here will most likely never be more than 2 decimals.